src/redis_ai_objects/tensor.c

```diff
@@ -359,10 +358,12 @@ RAI_Tensor *RAI_TensorCreateFromDLTensor(DLManagedTensor *dl_tensor) {
                                                   .byte_offset = dl_tensor->dl_tensor.byte_offset},
                           .manager_ctx = dl_tensor->manager_ctx,
                           .deleter = dl_tensor->deleter};
-
+    RAI_TensorLength(ret); // This will set the ret->len field.
```

Comment:
why the addition?

Reply:
To set the `len` field in `RAI_Tensor`. It is set to `ULONG_MAX` when calling 'RAI_TensorNew`